data/aliases.js

```diff
@@ -826,29 +826,801 @@ let BattleAliases = {
 	"zwater": "Hydro Vortex",
 
 	// Japanese names
-	"birijion": "Virizion",
-	"terakion": "Terrakion",
-	"agirudaa": "Accelgor",
-	"randorosu": "Landorus",
-	"urugamosu": "Volcarona",
-	"erufuun": "Whimsicott",
+	"fushigidane": "Bulbasaur",
+	"fushigisou": "Ivysaur",
+	"fushigibana": "Venusaur",
+	"hitokage": "Charmander",
+	"rizaado": "Charmeleon",
+	"rizaadon": "Charizard",
+	"zenigame": "Squirtle",
+	"kameiru": "Wartortle",
+	"kamekkusu": "Blastoise",
+	"kyatapi": "Caterpie",
+	"toranseru": "Metapod",
+	"batafuri": "Butterfree",
+	"bidoru": "Weedle",
+	"kokuun": "Kakuna",
+	"supiaa": "Beedrill",
+	"poppo": "Pidgey",
+	"pijon": "Pidgeotto",
+	"pijotto": "Pidgeot",
+	"koratta": "Rattata",
+	"ratta": "Raticate",
+	"onisuzume": "Spearow",
+	"onidoriru": "Fearow",
+	"bo": "Ekans",
+	"bokku": "Arbok",
```

Comment:
Arbok is `aabokku` in Japanese (official romaji `arbok`)

Reply:
Another case of an accent on the leading letter - this is easy to fix.